pom.xml

```diff
@@ -351,7 +351,7 @@
         </plugin>
         <plugin>
           <artifactId>maven-release-plugin</artifactId>
-          <version>2.5.3</version>
+          <version>3.0.0-M6</version>
```

Comment:
For consistency, the version of `maven-release-plugin` in https://github.com/jenkinsci/pom/blob/1279075509256ed961de63d26e72adc703da0ff6/pom.xml#L156= should be updated to the same version. Otherwise, the plugin toolchain will be using a different version from the core toolchain, which ultimately results in technical debt. An example of how such an inconsistency previously led to technical debt in the past is how when the plugin toolchain used a recent version of Surefire while the core toolchain used an old version it became a big challenge to upgrade Surefire in core in https://github.com/jenkinsci/jenkins/pull/6652, ultimately delaying the timely delivery of JUnit 5 support in https://github.com/jenkinsci/jenkins/pull/6657.